### ISSUE TITLE
Is it possible to run per application only single lambda?

### ISSUE THREAD

**Issue by riston** [kubek2k/lambdoku#11]
Is it only possible to run single lambda per application or for example multiple worker processes?

**Comment by kubek2k** [comment]
> Is it only possible to run single lambda per application or for example multiple worker processes?

@riston according to the model provided by AWS Lambda, you shouldn't really do that - but thats a good question - how to handle multiple lambdas coming from the same codebase for example.

**Comment by riston** [comment]
In AWS Lambda you "scale" only single function, in Heroku you could have multiple workers under single application and scale these separately `workerA=3dynos`, `workerB=1dyno` etc. Not sure if this would be some use case, this would add definetly complexity.

**Comment by kubek2k** [comment]
> In AWS Lambda you "scale" only single function, in Heroku you could have multiple workers under single application and scale these separately workerA=3dynos, workerB=1dyno etc. Not sure if this would be some use case, this would add definetly complexity.

yeah I got what you mean - in theory I could make `init` operation accept multiple arguments, that would make multiple lambdas default.
